libraries/AP_Notify/AP_BoardLED.cpp

```diff
@@ -98,7 +98,7 @@ void AP_BoardLED::update(void)
     static uint8_t arm_counter = 0;
 	if (AP_Notify::flags.armed) {
         // red led solid
-        hal.gpio->write(HAL_GPIO_A_LED_PIN, HAL_GPIO_LED_ON);
+//        hal.gpio->write(HAL_GPIO_A_LED_PIN, HAL_GPIO_LED_ON);
```

Comment:
Yes. This port is clearly only targetting revomini. Hence why it is failing in Travis